doc/flatpak-build-finish.xml

```diff
@@ -341,6 +341,8 @@ key=v1;v2;
 
                 <listitem><para>
                   Add extension point info.
+                  See the documentation for flatpak metadata for the possible values of
```

Comment:
It would be better to link to the manpage with `<citerefentry><refentrytitle>flatpak-metadata</refentrytitle><manvolnum>5</manvolnum></citerefentry>`

Reply:
Good suggestion, I was looking for the syntax for that but couldn't remember it. Fixed.